Code/GraphMol/MolAlign/testMolAlign.cpp

```diff
@@ -879,7 +897,11 @@ int main() {
   std::cout << "***********************************************************\n";
   std::cout << "Testing MolAlign\n";
 
-#if 1
+  std::cout << "\t---------------------------------\n";
+  std::cout << "\t test1AlignMetrics \n\n";
+  test1AlignMetrics();
+
+#if 0
```

Comment:
Please don't check in code that has tests that have been disabled like this. It's way too easy to forget to re-enable them later.